### DISCUSSION TITLE
How do I delete a context or a project?

### DISCUSSION THREAD

**Discussion by Grydpype** [ransome1/sleek#603] [General]
Dear all,
in the previous version of Sleek, there was a context menu under the right mouse button permitting me to rename or delete a project or a context.
This possibility no longer exists, at least in my current version 2.0.2 under linux.
So how do I do it now?

Kind regards,
kai.

PS- does Sleek not show the priorities of the tasks, or have I inadvertedly disabled this? If so, how do I turn them back on?
PPS - the archived todos are no longer stored in the directory of the todo.txt but rather somewhere like /run/user/... Why is this so; is this a bug or a feature?

**Comment by ransome1** [comment]
> So how do I do it now?

You currently don't, at least not with sleek. I recall digging into the tracking data, that some users provided in the old version of sleek. The feature at hand wasn't among what was frequently used so it got de-prioritized. At this point I don't have a precise plan on when this will be implemented. It is good, that you opened this discussion though, which proves there is some interest in it.

A workaround would of course be editing the todo.txt file in a text editor and search and replace the attributes you want to rename or remove.

> PS- does Sleek not show the priorities of the tasks, or have I inadvertedly disabled this? If so, how do I turn them back on?

Since the priorities are of course still available I can only guess what you are referring to here. I assume you mean the priority bubbles, when the 1st sorting group is not set to priority? That hasn't been implemented yet.

> PPS - the archived todos are no longer stored in the directory of the todo.txt but rather somewhere like /run/user/... Why is this so; is this a bug or a feature?

Are you using Flatpak? This might give you an explanation: https://github.com/ransome1/sleek/issues/564. In 2.0.3 the behavior of archiving will slightly change. Before 1st archiving you will need to select/create the archiving file, in order to provide a better UX in general and be more compliant with sandboxed environments such as Flatpak or macOS store releases.

**Reply by Grydpype** [reply]
Good morning,

thanks for the quick reply; that explains everything.
And I am in fact using Sleek via Flathub and #564 is my situation exactly; should have checked the discussions more carefully; sorry about that.

Thanks again for this slick software; I enjoy using it and it helps me immensely in my daily work.

Kind regards,
kai.
